src/devices/LiquidLevel/LiquidLevelSwitch.cs

```diff
@@ -22,12 +22,11 @@ public class LiquidLevelSwitch : IDisposable
         /// <summary>Creates a new instance of the LiquidLevelSwitch.</summary>
         /// <param name="dataPin">The data pin</param>
         /// <param name="liquidPresentPinState">Data pin state representing liquid being present</param>
-        /// <param name="pinNumberingScheme">Use the logical or physical pin layout</param>
         /// <param name="gpioController">A Gpio Controller if you want to use a specific one</param>
         /// <param name="shouldDispose">True to dispose the Gpio Controller</param>
-        public LiquidLevelSwitch(int dataPin, PinValue liquidPresentPinState, GpioController? gpioController = null, PinNumberingScheme pinNumberingScheme = PinNumberingScheme.Logical, bool shouldDispose = true)
+        public LiquidLevelSwitch(int dataPin, PinValue liquidPresentPinState, GpioController? gpioController = null, bool shouldDispose = true)
```

Comment:
since we are touching this we should consider using Pin class here given we're taking controller and a ping

Reply:
I think we should add those ctors, not replace them. And better in a separate PR.